### ISSUE TITLE
I get error in "onKey" thing

### ISSUE THREAD

**Issue by UserFucked** [HaqGamer/Chrome-Client#1]
@EventTarget
	public void onKey(EventKeyboard e) {
		MODULE_MANAGER.getMods().forEach(m -> {
			if(m.getKeyCode() == e.getKey())
				m.toggle();
		});
m.toggle(); Does NOT work.

**Comment by HaqGamer** [comment]
Are you use you are using java 8? And do you have a method called "toggle" in your module class?

**Comment by GlitchMasta47** [comment]
Have the same issue. No errors or warnings are shown, and I am running Java 8 (I had to in order to support the `m -> {etc.}` code). I do have the toggle code exactly as you wrote was in episode 2. In the game, I press the key and absolutely nothing happens. I tried switching from `Keyboard.KEY_F` to the integer counterpart of `33` but there was no difference. There isn't anything in the console that points to an error (nothing at all, in fact). Not sure what it could be. Maybe there is something with the Event register. Checking code still to find any issues...
